docs/tutorials/create_custom_identity.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: Create a Custom Identity
+parent: pages.tutorials
+nav_order: 8
+---
+
+# Create a Custom Identity
+
+{: .no_toc }
+
+## Table of contents
+
+{: .no_toc .text-delta }
+
+1. TOC
+   {:toc}
+
+---
+
+## Quick reference
+
+Out of the box, a FireFly Supernode contains both an `org` and a `node` identity. Your use case might demand more granular notions of identity (ex. customers, clients, etc.). Instead of creating a Supernode for each identity, you can create multiple custom identities within a FireFly Supernode.
+
+## Additional info
+
+- Reference: [Identities](../reference//identities.md)
+- Swagger: <a href="../swagger/swagger.html#/Default%20Namespace/postNewIdentity" data-proofer-ignore>POST /api/v1/identities</a>
+
+### Step 1: Create a signing key for new identity
```

Comment:
Think it's important to caveat that this is an instruction with the default CLI created environment, and creating signing keys for other more complex signing configuration is not covered in this doc.

Reply:
Also worth noting there is a CLI command for this, so you don't have to use an API

Reply:
Agreed with noting the caveats. Very important. However, the CLI command _does not_ actually invoke this API. The CLI command is used for creating the new private key / address, and installing the key on the signer - or creating the new identity via the Fabric CA if you're using Fabric.